sources/packages/backend/apps/queue-consumers/src/clam-antivirus.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from "@nestjs/common";
```

Comment:
Why are we having this clamav module inside the queue-consumers ? doesnt it need to be in a seperate folder structure? Please reverify the implementation.